src/main/java/com/booster/cliclient/BoosterCliLauncher.java

```diff
@@ -30,16 +30,16 @@ public void start() {
                     outputWriter.print("Unsupported command");
                     break;
                 case ADD_VOCABULARY_ENTRY:
-                    addVocabularyEntry();
+                    addVocabularyEntryHandler.handle();
```

Comment:
Please rename this and the rest of the handlers as follows: `addVocabularyEntryCommandHandler`